fdbcli/KillCommand.actor.cpp

```diff
@@ -0,0 +1,117 @@
+/*
+ * KillCommand.actor.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2021 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "fdbcli/fdbcli.actor.h"
+
+#include "fdbclient/FDBOptions.g.h"
+#include "fdbclient/IClientApi.h"
+#include "fdbclient/Knobs.h"
+
+#include "flow/Arena.h"
+#include "flow/FastRef.h"
+#include "flow/ThreadHelper.actor.h"
+#include "flow/actorcompiler.h" // This must be the last #include.
+
+namespace fdb_cli {
+
+ACTOR Future<bool> killCommandActor(Reference<IDatabase> db,
+                                    Reference<ITransaction> tr,
+                                    std::vector<StringRef> tokens,
+                                    std::map<Key, std::pair<Value, ClientLeaderRegInterface>>* address_interface) {
+	state bool result = true;
+	if (tokens.size() == 1) {
+		// Hold the reference to the standalone's memory
+		state ThreadFuture<RangeResult> kvsFuture =
+		    tr->getRange(KeyRangeRef(LiteralStringRef("\xff\xff/worker_interfaces/"),
```

Comment:
This code to read `\xff\xff/worker_interfaces/` is duplicated across a few commands it looks like